### PR TITLE
clean unused function

### PR DESCRIPTION
**Pull request by WanLinghao**
**What this PR does / why we need it**:
clean unused function
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/templates/templater.go [hunk 1]
```diff
@@ -234,15 +234,6 @@ func rpad(s string, padding int) string {
 	return fmt.Sprintf(template, s)
 }
 
-func indentLines(s string, indentation int) string {
```
Comment (nikhita) [inline pkg/kubectl/cmd/templates/templater.go]: :+1: thanks for the cleanup!

**Comment by nikhita** [comment]
/assign @soltysh 
for approval

**Comment by WanLinghao** [comment]
@janetkuo PTAL

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**